src/haddock/libs/libparallel.py

```diff
@@ -82,7 +80,11 @@ def num_processes(self):
 
     @num_processes.setter
     def num_processes(self, n):
-        self._ncores = parse_ncores(n)
+        self._ncores = parse_ncores(
```

Comment:
Need to test this to ensure it's working properly. It should unless some unexpected.